src/routes/users_routes.js

```diff
@@ -1,7 +1,10 @@
 const app = require("express").Router();
 const usersController = require("../controllers/users_controller");
+const config = require("../../config");
```

Comment:
You can destructure the config object so you just have env

Reply:
true